File: Java/Fact.java
import java.util.Scanner;

public class fact {
    public static void main(String[] args) {
        int num,fact=1,i;
        System.out.print("Enter the number: ");
        Scanner n=new Scanner(System.in);
        num=n.nextInt();
        for (i=1;i<=num;i++)
        {
            fact*=i;
        }
        System.out.printf("The factorial of %d is %d",num,fact);
    }
    
}

File: C/final/fact.c
#include <stdio.h>
void main()
{
    int fact=1,num,i;
    printf("Enter the number: ");
    scanf("%d",&num);
    for (i=1;i<=num;i++)
    {fact=fact*i;
    }
    printf("factorial of %d is %d",num,fact);
}